requirements.txt

```diff
@@ -1,9 +1,19 @@
+# for database
+-r db/requirements.txt
+
+# for onboarding
+-r onboarding/requirements.txt
+
+# for Rasa agent
 -r Rasa_Bot/requirements.txt
-alembic
-connexion
+
+# for rasa-scheduler
+-r requirements-scheduler.txt
+
+# for sensor API
+-r sensor_api/requirements.txt
+
 requests==2.25.1
 pandas_ods_reader==0.0.7
 git+https://github.com/PerfectFit-project/pa-algorithms
 
-# for rasa-scheduler
--r requirements-scheduler.txt
```

Comment:
These comments are all redundant, it is clear from the namespace of the requirement files which component it belongs to!